docs/src/content/docs/fr/reference/configuration.mdx

```diff
@@ -346,6 +345,72 @@ starlight({
 });
 ```
 
+### `expressiveCode`
+
+**Type :** `StarlightExpressiveCodeOptions | boolean`  
+**Par défaut :** `true`
+
+Starlight utilise [Expressive Code](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) pour afficher les blocs de code et ajouter le support pour mettre en évidence des portions d'exemples de code, ajouter des noms de fichiers aux blocs de code, et plus encore.
+Consultez le [guide « Blocs de code »](/fr/guides/authoring-content/#blocs-de-code) pour apprendre à utiliser la syntaxe d'Expressive Code dans votre contenu Markdown et MDX.
```

Comment:
The link is currently broken in this PR, I'll update this comment with a reference to the PR fixing it.

Reply:
Updated the branch now that #1118 has been merged and the link is working properly in the preview.